www/js/app.js

```diff
@@ -17,4 +17,9 @@ $(document).ready(function() {
 		advSearch.slideToggle(0);
 		return false;
 	});
+
+	$('.invert-selection').on('click', function() {
+		$(this).parent().find('input').click();
+		return false;
+	});
```

Comment:
Jinak pokud bychom se bavili čistě o kódu, tak lépe tohle je zbytečně těsně vázané na aktuální strukturu HTML a formuláře obecně, navíc to spouští události nad všemi inputy. Lépe tedy např.

~~~js
$(this).closest('form').find('.checkbox').find(':checkbox').click();
~~~